viewing_party/main.py

```diff
@@ -0,0 +1,165 @@
+def create_movie(title, genre, rating):
+      # if all three parameters are truthy, return a dict with key val pairs
+       #return a dictionary with 3 keys title, genre, rating 
+    #else title, genre, or rating is falsey
+        #return None
+    movie_dict = {}
+    movie_args = []
+    movie_args.append(title)
+    movie_args.append(genre)
+    movie_args.append(rating)
+    for i in movie_args:
+        if not i:
+            return None
```

Comment:
Variable names like `i` are ok for counter variables, but try to provide more meaningful variable names for other use cases.  In this case you might consider naming it `arg`